astroquery/casda/tests/test_casda_remote.py

```diff
@@ -33,6 +34,22 @@ def test_query_region_text_radius(self):
         responses = Casda.query_region('22h15m38.2s -45d50m30.5s', radius=0.5 * u.arcmin, cache=False)
         assert isinstance(responses, Table)
         assert len(responses) > 1
-        print(responses[0])
+
         for key in ('dataproduct_type', 'obs_id', 'access_url', 'access_format', 'obs_release_date'):
             assert key in responses.keys()
+
+    @pytest.mark.skipif(('CASDA_USER' not in os.environ or
+                        'CASDA_PASSWD' not in os.environ),
+                        reason='Requires real CASDA user/password (CASDA_USER '
```

Comment:
is there any dummy user/passwd that we could use for testing? I don't do that for most other modules, but would be nice it it were possible (even if it only serves some very limited amount test data). 
@keflavich - I think this is a topics for the future we may want to raise with multiple archives, I'm certain there is a way to store the secrets somewhere safe for CI (if not for travis, the for github actions).